src/components/Accordion/Accordion.module.scss

```diff
@@ -80,65 +80,71 @@ $group-border-radius: 0.5rem;
     }
 
     &:active {
-      background-color: #{$purple-2};
+      background-color: $color-primary-2;
+      border-bottom: 0.063rem solid $color-primary-7;
     }
   }
 
   &[data-contrast='high'] {
-    .Content {
-      border: 0.063rem solid $purple-6;
-    }
+    $hc-background: $color-primary-2;
+    $hc-trigger-border: $color-primary-5;
+    $hc-trigger-border-hover: $color-primary-8;
 
-    .Header {
-      border-bottom: 0.063rem solid #{$purple-6};
-
-      &[data-state='open'] {
-        border-bottom: none;
-      }
+    .Trigger {
+      background-color: $hc-background;
+      padding: 0.5rem 1rem 0.438rem;
+      border-bottom: 0.063rem solid $hc-trigger-border;
 
       &:hover {
-        border-bottom: 0.063rem solid #{$purple-8};
+        background-color: $hc-background;
+        border-bottom: 0.063rem solid $hc-trigger-border-hover;
       }
 
       &:active {
-        border-bottom: 0.063rem solid #{$purple-6};
+        border-bottom: 0.063rem solid $hc-trigger-border;
       }
     }
 
-    .Trigger {
-      background-color: #{$purple-4};
-      padding: 0.5rem 1rem 0.438rem;
-
-      &[data-state='open'] {
-        background-color: #{$purple-2};
-      }
-
-      &:hover {
-        background-color: #{$purple-2};
-      }
+    .Content::before {
+      border: 1px solid $color-primary-6;
```

Comment:
should we replace `px` with `0.063rem`

Reply:
Good question - imo it makes sense to keep borders `1px` so they don't scale up, but perhaps this is incorrect. I have pinged design team!